### ISSUE TITLE
Arduino Nano IoT

### ISSUE THREAD

**Issue by LukMas** [arpruss/ADCTouchSensor#5]
Hi! 

I've ordered an Arduino Nano IoT, planning to use it with the CapSense library. But now I've found this one. It looks more reliable in the sensing than the other. I would give it a try, but I don't know if it works with the board. It comes with an ARM SAMD21 Cortex®-M0+ 32bit. It belongs to the family STM32 32-bit Arm Cortex MCUs, part of the Ultra Low Power chips. 

Is this library ready also for this chip? If not, what should (could) I do to make it working?

**Comment by arpruss** [comment]
No, the library isn't for it. It MIGHT work with some minor changes, especially if you are able to ground one analog pin.
